Dalamud/Game/Addon/Lifecycle/AddonLifecycle.cs

```diff
@@ -197,7 +208,14 @@ private void OnAddonSetup(AtkUnitBase* addon, uint valueCount, AtkValue* values)
             Log.Error(e, "Exception in OnAddonSetup pre-setup invoke.");
         }
 
-        addon->OnSetup(valueCount, values);
+        try
+        {
+            addon->OnSetup(valueCount, values);
+        }
+        catch (Exception e)
+        {
+            Log.Error(e, "Caught exception when calling original AddonSetup. This may be a bug in the game or another plugin hooking this method.");
```

Comment:
(Not a review - just idle commentary)

Does it make sense to handle exceptions calling original? If the game is breaking things, we should let it crash imo. I'm unsure if it's additionally worth bailing out problems caused by other plugin devs - they'd crash the game if this system didn't exist, or if they manage to get their hooks in first somehow.